kuma/wiki/jinja2/wiki/document.html

```diff
@@ -301,7 +306,7 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
         (function($) {
             $('#wikiArticle').children('h2[id]').each(function() {
                 var id = $(this).attr('id');
-                var href = '{{ document.get_edit_url() }}#' + id;
+                var href = '{{ doc_edit_url }}#' + id;
```

Comment:
again, nice catch! I'm not sure section editing will survive going through login, but I think we only show it for some logged in users. Maybe no users? My memory is failing me.